torchvision/models/vision_transformer.py

```diff
@@ -565,8 +592,17 @@ def vit_l_32(*, weights: Optional[ViT_L_32_Weights] = None, progress: bool = Tru
     `"An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale" <https://arxiv.org/abs/2010.11929>`_.
```

Comment:
I would be in favour of removing the quotes when linking to the paper, because the link highlighting gives a clear separation already. Would you mind removing the quotes in the links for ViT and RegNet as well?

Reply:
sure